docs/en/quick-answers/troubleshooting/context-missing-task-step.md

```diff
@@ -1,12 +1,31 @@
 ---
 layout: default
-title: My context is missing the Task/Step but it exists as part of the filename.
+title: Why is my context missing the Task/Step when it exists as part of the filename?
 pagename: context-missing-task-step
 lang: en
 ---
 
-# My context is missing the Task/Step but it exists as part of the filename.
+# Why is my context missing the Task/Step when it exists as part of the filename?
 
-Toolkit uses the path cache to help associate paths on disk with entities in Shotgun and build your context. Since the path cache only stores **folder** information and not filenames, any context-related information contained in the filename (including Task or Step name) will not be detected when building the context.
+When you create folders via Toolkit it registers the path against the entity so that a lookup may be performed. This means that given a path, you can determine the correct context.
+Toolkit will only create registries for folders generated from the schema, and so it doesn't consider things like file names or folders that were defined solely in the `templates.yml`.
```

Comment:
> solely in the `templates.yml`.

should be "solely in the `templates.yml` file."

Reply:
done!